db/db_test2.cc

```diff
@@ -5605,6 +5675,7 @@ TEST_F(DBTest2, ChangePrefixExtractor) {
     }
     {
       std::unique_ptr<Iterator> iterator(db_->NewIterator(ro));
+      ASSERT_OK(iterator->status());
```

Comment:
Please move to right before/after Valid check.

Reply:
Done.